osu.Game/Screens/Play/Player.cs

```diff
@@ -564,6 +573,8 @@ public void Pause()
             GameplayClockContainer.Stop();
             PauseOverlay.Show();
             lastPauseActionTime = GameplayClockContainer.GameplayClock.CurrentTime;
+
+            samplePause?.Play();
```

Comment:
Related to issue I mentioned about resuming - this can play multiple times if you go into the resume sequence and then pause again, keeping one sample in a playing state in the background while the other is being stopped. Would also be fixed by moving the stop out of the callback.